application/src/main/java/io/redlink/smarti/webservice/pojo/A_SourceConfiguration.java

```diff
@@ -0,0 +1,33 @@
+package io.redlink.smarti.webservice.pojo;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class A_SourceConfiguration implements I_SourceConfiguration {
+	
+	@Override
+	public JSONObject asJSON() {
+
+		try {
+			return (JSONObject) new JSONParser().parse(new ObjectMapper().writeValueAsString(this));
```

Comment:
Why do you need this? As far as I can see you always use the toJsonString method on things produces by this method. So a direct serialization with ObjectMapper is more straight forward.